sqlglot/dialects/presto.py

```diff
@@ -276,7 +276,7 @@ class Parser(parser.Parser):
                 this=seq_get(args, 0), substr=seq_get(args, 1), instance=seq_get(args, 2)
             ),
             "TO_CHAR": _build_to_char,
-            "TO_HEX": exp.Hex.from_arg_list,
+            "TO_HEX": exp.UpperHex.from_arg_list,
```

Comment:
Why are we using `UpperHex` and not `Hex` by default if the base class defaults to `HEX_LOWERCASE = False`? This is inconsistent right? I'd expect `Hex` to just be treated as "upper hex".

Reply:
Thank you. So Hex should be treated as "upper hex" becasue the base `HEX_LOWERCASE = False`. And we are going to create a new LowerHex instead of UpperHex expression. I will update the PR.